web-storage/README.md

```diff
@@ -0,0 +1,47 @@
+# Text
+
+Web Storage provides mechanisms to store key value pairs in a much more intuitive way than using cookies.
```

Comment:
Please add one or two more sentences here about web storage and why it's different from cookies.